.eslintrc.js

```diff
@@ -3,17 +3,13 @@
  */
 module.exports = {
   root: true,
-  extends: [
-    'eslint-config-digitalbazaar',
-    // 'eslint-config-digitalbazaar/jsdoc'
-  ],
   env: {
     node: true
   },
-  parserOptions: {
-    // this is required for dynamic import()
-    ecmaVersion: 2020
-  },
+  extends: [
+    'digitalbazaar',
+    // 'digitalbazaar/jsdoc'
```

Comment:
Intentionally commented out?

Reply:
It was already commented out.  233 issues need addressing.  Will leave that for another time.